### ISSUE TITLE
Fixes for accessibility

### ISSUE THREAD

**Issue by jdirand** [InseeFr/Lunatic#730]
- [x] If there is an error on a component, it should have the attribute aria-invalid="true" #731
- [x] If a component has an aria-invalid="true", we should set the autofocus on that component. If there are multiple components on a page with aria-invalid="true", then the first component with this attibute aria-invalid="true" should be focused. #734
- [x] The cleaning and resizing should call on handleChange #732
- [x] We should remove the min, max and step attributes on the input number that use an input #733
